libs/features/placeholders.js

```diff
@@ -10,12 +10,16 @@ const getPlaceholdersPath = (config, sheet) => {
 };
 
 const parsePlaceholderJson = async (resp, placeholders) => {
-  const json = resp.ok ? await resp.json() : { data: [] };
-  if (!json.data.length) return;
-  json.data.forEach((item) => {
-    window.mph[item.key] = item.value;
-    placeholders[item.key] = item.value;
-  });
+  try {
+    const json = resp.ok ? await resp.json() : { data: [] };
+    if (!json.data.length) return;
+    json.data.forEach((item) => {
```

Comment:
Tiny nit, since this file is used by everyone:
```suggestion
    json.data?.forEach((item) => {
```
If `json.data` is an empty array, then `forEach` wouldn't be executed. Saves one line of code, but not the end of the world if it stays like this either.